### ISSUE TITLE
Option to include sprite size in generated css

### ISSUE THREAD

**Issue by F21** [glebmachine/postcss-easysprites#11]
If I have the following css:

``` css
.facebook{
    background-image: url('./test.png#sprites');
}
```

The following is generated:

``` css
.facebook{
    background-image: url(51e83fd2b1c443620524b9c2c8fa29db.png); background-position: -1707px 0;
}
```

However, to get it to display correctly, I need to manually add the size of the sprite into the original css:

``` css
.facebook{
    background-image: url('./test.png#sprites');
    width: 100px;
    height: 100px;
}
```

It would be nice if the generated css output automatically included the size (via an option):

``` css
.facebook{
    background-image: url(51e83fd2b1c443620524b9c2c8fa29db.png); background-position: -1707px 0;
    width: 100px;
    height: 100px;
}
```

**Comment by glebmachine** [comment]
Yep, you should to handle it by youself.

Let your preprocessors do this job for you)

Sass/compass mixin example
```sass
=imgbox($img)
  & #{$selector}
    width : image-width($img)
    height : image-height($img)
    background-image : image-url($img)
    background-size : contain
    background-repeat : no-repeat
```

**Comment by F21** [comment]
I want to avoid compass though (after using compass/sass, I find postcss and cssnext + plugins to be much easier to use).

I have looked at a few webpack and postcss plugins, but they either seem quite buggy, or don't work.

**Comment by F21** [comment]
I just tried postcss-assets, and it works! However, when I use it with postcss-easysprites, postcss-easysprites crashes.

The trick to fix it is to include `postcss-easysprites` after `postcss-assets`:
```
postcss: function () {
        return [

            require('postcss-assets')(),
            require('postcss-easysprites')()
        ];
    }
```

Can you look at why this is the case? If `postcss-easysprites` is included before `postcss-assets`, I get:
```
ERROR in ./src/sprite-test.css
Module build failed: ModuleBuildError: Module build failed: TypeError: Cannot read property 'input' of undefined
    at C:\Work\test\webpack-postcss-easysprites\node_modules\postcss-assets\index.js:222:33
    at C:\Work\test\webpack-postcss-easysprites\node_modules\postcss\lib\container.js:88:34
    at C:\Work\test\webpack-postcss-easysprites\node_modules\postcss\lib\container.js:73:26
    at Rule.each (C:\Work\test\webpack-postcss-easysprites\node_modules\postcss\lib\container.js:60:22)
    at Rule.walk (C:\Work\test\webpack-postcss-easysprites\node_modules\postcss\lib\container.js:72:21)
    at C:\Work\test\webpack-postcss-easysprites\node_modules\postcss\lib\container.js:76:32
    at Root.each (C:\Work\test\webpack-postcss-easysprites\node_modules\postcss\lib\container.js:60:22)
    at Root.walk (C:\Work\test\webpack-postcss-easysprites\node_modules\postcss\lib\container.js:72:21)
    at Root.walkDecls (C:\Work\test\webpack-postcss-easysprites\node_modules\postcss\lib\container.js:86:25)
    at Assets.postcss (C:\Work\test\webpack-postcss-easysprites\node_modules\postcss-assets\index.js:219:7)
    at LazyResult.run (C:\Work\test\webpack-postcss-easysprites\node_modules\postcss\lib\lazy-result.js:206:20)
    at C:\Work\test\webpack-postcss-easysprites\node_modules\postcss\lib\lazy-result.js:120:37
    at LazyResult.asyncTick (C:\Work\test\webpack-postcss-easysprites\node_modules\postcss\lib\lazy-result.js:134:15)
    at C:\Work\test\webpack-postcss-easysprites\node_modules\postcss\lib\lazy-result.js:125:31
    at _fulfilled (C:\Work\test\webpack-postcss-easysprites\node_modules\q\q.js:834:54)
    at self.promiseDispatch.done (C:\Work\test\webpack-postcss-easysprites\node_modules\q\q.js:863:30)
    at Promise.promise.promiseDispatch (C:\Work\test\webpack-postcss-easysprites\node_modules\q\q.js:796:13)
    at C:\Work\test\webpack-postcss-easysprites\node_modules\q\q.js:556:49
    at runSingle (C:\Work\test\webpack-postcss-easysprites\node_modules\q\q.js:137:13)
    at flush (C:\Work\test\webpack-postcss-easysprites\node_modules\q\q.js:125:13)
    at nextTickCallbackWith0Args (node.js:452:9)
    at process._tickCallback (node.js:381:13)
    at DependenciesBlock.onModuleBuildFailed (C:\Work\test\webpack-postcss-easysprites\node_modules\webpack-core\lib\NormalModuleMixin.js:315:19)
    at nextLoader (C:\Work\test\webpack-postcss-easysprites\node_modules\webpack-core\lib\NormalModuleMixin.js:270:31)
    at C:\Work\test\webpack-postcss-easysprites\node_modules\webpack-core\lib\NormalModuleMixin.js:292:15
    at context.callback (C:\Work\test\webpack-postcss-easysprites\node_modules\webpack-core\lib\NormalModuleMixin.js:148:14)
    at C:\Work\test\webpack-postcss-easysprites\node_modules\postcss-loader\index.js:58:17
    at process._tickCallback (node.js:401:9)
 @ ./src/app.js 1:0-28
Child html-webpack-plugin for "index.html":
        + 1 hidden modules
Child extract-text-webpack-plugin:
        + 1 hidden modules

    ERROR in ./~/css-loader?sourceMap&importLoaders=1!./~/postcss-loader!./src/sprite-test.css
    Module build failed: TypeError: Cannot read property 'input' of undefined
        at C:\Work\test\webpack-postcss-easysprites\node_modules\postcss-assets\index.js:222:33
        at C:\Work\test\webpack-postcss-easysprites\node_modules\postcss\lib\container.js:88:34
        at C:\Work\test\webpack-postcss-easysprites\node_modules\postcss\lib\container.js:73:26
        at Rule.each (C:\Work\test\webpack-postcss-easysprites\node_modules\postcss\lib\container.js:60:22)
        at Rule.walk (C:\Work\test\webpack-postcss-easysprites\node_modules\postcss\lib\container.js:72:21)
        at C:\Work\test\webpack-postcss-easysprites\node_modules\postcss\lib\container.js:76:32
        at Root.each (C:\Work\test\webpack-postcss-easysprites\node_modules\postcss\lib\container.js:60:22)
        at Root.walk (C:\Work\test\webpack-postcss-easysprites\node_modules\postcss\lib\container.js:72:21)
        at Root.walkDecls (C:\Work\test\webpack-postcss-easysprites\node_modules\postcss\lib\container.js:86:25)
        at Assets.postcss (C:\Work\test\webpack-postcss-easysprites\node_modules\postcss-assets\index.js:219:7)
        at LazyResult.run (C:\Work\test\webpack-postcss-easysprites\node_modules\postcss\lib\lazy-result.js:206:20)
        at C:\Work\test\webpack-postcss-easysprites\node_modules\postcss\lib\lazy-result.js:120:37
        at LazyResult.asyncTick (C:\Work\test\webpack-postcss-easysprites\node_modules\postcss\lib\lazy-result.js:134:15)
        at C:\Work\test\webpack-postcss-easysprites\node_modules\postcss\lib\lazy-result.js:125:31
        at _fulfilled (C:\Work\test\webpack-postcss-easysprites\node_modules\q\q.js:834:54)
        at self.promiseDispatch.done (C:\Work\test\webpack-postcss-easysprites\node_modules\q\q.js:863:30)
        at Promise.promise.promiseDispatch (C:\Work\test\webpack-postcss-easysprites\node_modules\q\q.js:796:13)
        at C:\Work\test\webpack-postcss-easysprites\node_modules\q\q.js:556:49
        at runSingle (C:\Work\test\webpack-postcss-easysprites\node_modules\q\q.js:137:13)
        at flush (C:\Work\test\webpack-postcss-easysprites\node_modules\q\q.js:125:13)
        at nextTickCallbackWith0Args (node.js:452:9)
        at process._tickCallback (node.js:381:13)
```

You can see it in the `crash-postcss-assets` branch: https://github.com/F21/webpack-postcss-easysprites/tree/crash-postcss-assets

**Comment by glebmachine** [comment]
Ok, thank you, will check it soon)

**Comment by patrickcate** [comment]
@F21 this option is now included with version 1.0.0.
